sdk/internal/recording/recording.go

```diff
@@ -606,13 +188,14 @@ func Start(t *testing.T, pathToRecordings string, options *RecordingOptions) err
 		return err
 	}
 	recId := resp.Header.Get(IDHeader)
+	log.Printf("test name: %s\t recording ID: %s\n", t.Name(), recId)
```

Comment:
This makes test runs spammy, can we make it optional or filterable via `internal/log`?